task/buildah-pr/0.1/buildah-pr.yaml

```diff
@@ -42,11 +45,13 @@ spec:
     outputs:
       - name: image
         type: image
-
+  workspaces:
+    - name: source
+      mountPath: /workspace/source
   steps:
     - name: build
       image: $(params.BUILDER_IMAGE)
-      workingDir: /workspace/source
+      workingDir: $(workspaces.source.path)
```

Comment:
This shouldn't be changed in the `-pr` task. (or maybe using `$(resources.inputs.source.path)` if it works)

Reply:
Done, replaced with `$(resources.inputs.source.path)`